apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -209,7 +209,9 @@ class TopInstructions extends Component {
    * contents of the comment tab.
    */
   forceTabResizeToMaxHeight = () => {
-    this.props.setInstructionsRenderedHeight(this.adjustMaxNeededHeight());
+    if (this.state.tabSelected === TabType.COMMENTS) {
```

Comment:
i think rather than checking `this.state.tabSelected === TabType.COMMENTS` here, we should instead not be calling `forceTabResizeToMaxHeight` unless we are on the comments tab. i know you're trying to get this out for the deploy, so non-blocking, but should be refactored later

Reply:
Ok I will refactor tomorrow.

Reply:
After discussion, this is the best way to do this for now so not refactoring